### ISSUE TITLE
unpublish resource type & delete resource type

### ISSUE THREAD

**Issue by yblatti** [easySDI/easySDI#717]
**Original author :** Portier Thomas

unpublish ressource type by clicking on the publishing icon in the ressource type list return the error : 
Fatal error: Call to a member function publish() on a non-object in /home/.../libraries/legacy/controller/admin.php on line 208

**Comment by yblatti** [comment]
**Original author :** Portier Thomas

This bug appears in the back-end -> EasySDI -> Catalog -> Resource Type

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

For info, this defect is still here in 4.2.0-beta-2.

**Comment by yblatti** [comment]
**Original author :** Mérour Xavier

Problem still there... and same behaviour if I want to delete the resource : 

```

Fatal error: Call to a member function publish() on a non-object in /home/data/www/www.asitvd.ch/html/libraries/legacy/controller/admin.php on line 208
```



This should have been corrected since a while (ticket is more than one year old...)

**Comment by yblatti** [comment]
**Original author :** Blatti Yves

Fixed by Defect #703
